app.py

```diff
@@ -283,11 +283,28 @@ def viewMovie(id):
 @api.route('/add/<int:movie_id>', methods=["POST", "GET"])
 @login_required
 def addMovie(movie_id: int):
-    print("hello")
-    # current_user.add_favorite_movie(movie_id)
     movie = get_movie_info(movie_id)
-    print(movie["title"])
-
+    id = movie_id
```

Comment:
is it necessary to save the dictionary values as variables instead of referencing them directly?